rpc/handlers.go

```diff
@@ -1957,22 +2007,22 @@ func (h *Handler) LegacyMethods() ([]jsonrpc.Method, string) { //nolint: funlen
 		{
 			Name:    "starknet_getBlockWithTxHashes",
 			Params:  []jsonrpc.Parameter{{Name: "block_id"}},
-			Handler: h.LegacyBlockWithTxHashes,
```

Comment:
We're not using Legacy methods anymore, right? Should we remove them?

Reply:
Yep, I'll remove them in upcoming PR. I'm a little bit worried that there are a bunch of Legacy methods already :D

Reply:
Want to double check that removing them doesn't affect anything